app.js

```diff
@@ -0,0 +1,158 @@
+//  in this file create an express application - use the middle-ware built into express
+//  to serve up static files from the public directory (index.html and client.js - you 
+//  can also serve up css files from the public directory this way if you want)
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+/*
+
+*/
+var express = require('express');  //require the express module which returns an object/function
+var app = express(); //creates and express application instance
+var bodyParser = require('body-parser');
+var truckRoutes = require('./routes/truckRoutes');
+var mongoose = require('mongoose');
+
+
+var serveStatic = express.static('public');
+app.use(serveStatic);
+app.use(bodyParser.urlencoded({extended: true}));
+app.use('/trucks', truckRoutes);
+
+//var db = mongoose.connect('mongodb://localhost/truckAPI');
```

Comment:
Need this line. The connection string should be 'mongodb://localhost/foodTruckAPI'.